docs/manual/cli/resize-images.de.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "contao:resize-images"
+description: "Process deferred images that were not resized."
+aliases:
+    - /de/cli/resize-images/
+weight: 30
+---
+
+
+{{< version "4.8" >}}
+
+Mit diesem Befehl können alle Bilder bearbeitet werden, deren Größe noch nicht geändert wurde.
```

Comment:
The German version is not very accurate (the English version is correct). May be something like this?

```suggestion
Mit diesem Befehl können alle fehlenden, verzögert erzeugten Bilder erstellt werden.
```

This would warrant an article about deferred images - but that's for a future PR.